code/api/tests/test_inbox.py

```diff
@@ -113,7 +119,7 @@ def test_post_local_post(self):
         inbox_post = author2.inbox_posts.first()
         self.assertEqual(inbox_post.title, dummy_post.title)
         self.assertEqual(inbox_post.description, dummy_post.description)
-        self.assertEqual(inbox_post.content, dummy_post.content)
+        self.assertEqual(inbox_post.decoded_content, dummy_post.decoded_content)
```

Comment:
This line failed in tests. See other comment

Reply:
Sorry, I noticed the issue with the tests and was going to fix in a commit, together with the buggy js code for the form. But, I can't seem to find the root cause of the latter bug.

Thanks a lot for the comments!

Still working on the buggy js, but I'm going to focus on some of my other assigned tasks a bit more.